core/src/avm2/globals/flash/display/NativeMenu.as

```diff
@@ -1,6 +1,141 @@
-package flash.display {
+package flash.display
+{
+
     import flash.events.EventDispatcher;
-    public class NativeMenu extends EventDispatcher {
+    import __ruffle__.stub_method;
+    import __ruffle__.stub_getter;
+    
+    [API("661")]
```

Comment:
`airglobal.swc` disagrees with the docs yet again - it seems like this is also technically versioned at `667`.

Reply:
If this is 667, then NativeMenuItem gotta be too, no?

Reply:
Yes, this is what I did